.editorconfig

```diff
@@ -0,0 +1,17 @@
+root = true
+
+[*]
+trim_trailing_whitespace = true
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+indent_style = space
+indent_size = 2
+max_line_length = 80
+
+[*.py]
+indent_style = space
+indent_size = 4
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
Just curious, why don't you trim trailing whitespaces in .md? Are there cases where it shouldn't be removed?

Reply:
@gimite removing trailing white space from .md can cause to break the formatting after rendering the markdown because markdown uses 4 spaces to indicate a new line and white spaces are  part of the language itself.

Reply:
I don't think we use spaces-for-newlines anywhere. Is there ever a reason to prefer 2/4 spaces for newlines instead of newlines or a backslash? The advantage of those options is it's more readable if you're looking at the raw .md file, since you wouldn't see the trailing whitespace. (Or, is there anywhere else Markdown considers trailing whitespace meaningful?)

Reply:
@nworden I'm not suggesting to use spaces, but removing trailing whitespace from markdowns files limits the language itself. 
I also think it's more readable if we look at the raw .me file without trailing whitespace.
Do you want me to remove the `trim_trailing_whitespace`?

Reply:
@mohitkyadav Can you maybe give us an example Markdown where it makes difference by removing trailing whitespace?

Reply:
Here's the screenshot of trimmed whitespace and with whitespace.
![without trimmed](https://user-images.githubusercontent.com/25580776/52997356-58949b80-3446-11e9-96d6-52f56c1256f5.jpg)
![trimmed](https://user-images.githubusercontent.com/25580776/52997354-57fc0500-3446-11e9-8da8-a8f819c4a91a.jpg)

Reply:
I see. I believe what @nworden means is that we won't write in that way, and instead we should write like this:
```md
Astralis wins \
NaVi wins
```

So it should be safe to trim trailing whitespaces.

Reply:
No problem I'll remove that.